share/spice/playing_cards/playing_cards.js

```diff
@@ -0,0 +1,29 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_playing_cards = function(api_result){
+        if(api_result.error || !api_result.cards) {
+            return Spice.failed('playing_cards');
+        }
+        
+        Spice.add({
+            id: "playing_cards",
+            name: "Deck of Cards",
+            data: api_result.cards,
+            meta: {
+                sourceName: "DeckOfCards",
+                sourceUrl: 'http://deckofcardsapi.com'
+            },
+            normalize: function(item) {
+                return {
+                    image: item.image
+                };
+            },
+            templates: {
+                group: 'movies',
```

Comment:
Let's switch this to `media`, then we can add a css file in the share dir and alter the tile and image as follows:

- `img` should have a height of 14em (this might change)
- the tile should have an `auto` width
- the img wrap should have an `auto` height

I believe that makes it so the image controls the height/width of the tile and that way we aren't stretching the image to fill the tile